src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,15 +1,99 @@
-import { Component } from 'react';
+import React from 'react';
 
-type Props = {};
-type State = {};
+import './NewMovie.scss';
 
-export class NewMovie extends Component<Props, State> {
-  state: State = {};
+type Props = {
+  onAdd: (movie: Movie) => void;
+};
+
+type State = {
+  title?: string;
+  imgUrl?: string;
+  imdbUrl?: string;
+  imdbId?: string;
+  description?: string;
```

Comment:
do not cheat with TS)

```suggestion
  title: string;
  imgUrl: string;
  imdbUrl: string;
  imdbId: string;
  description: string;
```